File: BackendDBD/src/main/java/uni/edu/pe/modulo_crm/dto/Mantenimientodto/Equipo.java
package uni.edu.pe.modulo_crm.dto.Mantenimientodto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Equipo {
    private String idEquipo;
    private String nombre;
    private String tipo;
    private String marca;
    private String modelo;
    private String serie;
    private Date fechaAdquisicion;
    private String estado;
    private String ubicacion;
    private String frecuencia;
    private String descripcion;
    private String ultimaRevision;
    private String proximaRevision;
}

File: FrontendDBD/frontend/src/app/models/modelsmantenimiento/equipo.ts
import { DatePipe } from "@angular/common";

export interface Equipo {
    idEquipo: string;
    nombre: string;
    tipo: string;
    marca?: string;
    modelo?: string;
    serie?: string;
    ubicacion?: string;
    fechaAdquisicion: Date | null;
    estado: string;
    frecuencia?: number;
    descripcion?: string;
    ultimaRevision: Date | null;
    proximaRevision: Date | null;

}
